### PR TITLE
Follow up #11

### PR DESCRIPTION
**Pull request by recena**
Follow up https://github.com/jenkinsci/external-monitor-job-plugin/pull/11

@reviewbybees especially @andresrc, @kwhetstone

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -35,16 +35,16 @@
   <licenses>
     <license>
       <name>The MIT license</name>
-      <url>https://www.opensource.org/licenses/mit-license.php</url>
+      <url>http://www.opensource.org/licenses/mit-license.php</url>
```
Comment (recena) [inline pom.xml]: The URL does not exist.

**Comment by raul-arabaolaza** [comment]
:bee:

**Comment by andresrc** [comment]
:bee:

**Comment by recena** [comment]
@reviewbybees done

**Pull request merged**